examples/design-system/packages/eslint-preset-acme/README.md

```diff
@@ -0,0 +1,3 @@
+# `scripts`
```

Comment:
Shouldn’t this be eslint-preset-acme instead?

Reply:
Yes